gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/types/foil-presentation.type.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: not sure if this is an exhaustive list, but it's what I got from jcard.js:createCardDiv
```

Comment:
yea, we will want to have the list of possible options defined because we will use this one list to power all refs to it through the client. I'll leave this for now, it won't cause issues.... but i will leave a note for future us when we are looking into why a foil type isn't working :)